### DISCUSSION TITLE
Volume not Working over WebInterface or GPIO

### DISCUSSION THREAD

**Discussion by Map3GOSM** [MiczFlor/RPi-Jukebox-RFID#2205] [Q&A] [answered]
Hi
I am having trouble with the Volume control of my Phoniebox. Audio is Playing via RFID or Webinterface but the volume is staying constant. I cant change it on the webInterface or the using the GPIO connected buttons. All other controls (Play Pause etc.) work via Web and buttons. I am Running on a Pi Zero WH with Bullseye. 

I updated form 2.4 to 2.5 this Morning that did not solve the Issue but I did notice the following line in the output during the update. 

```
volume: n/a   repeat: off   random: off   single: off   consume: off
```

That does not look right to me but I don't know what to do with that. 😊 

Here is my Global.conf file. Let me know if anything else would be of use to track down the cause.

```
AUDIOFOLDERSPATH="/home/pi/RPi-Jukebox-RFID/shared/audiofolders"
PLAYLISTSFOLDERPATH="/home/pi/RPi-Jukebox-RFID/playlists"
SWIPEORPLACE="SWIPENOTPLACE"
SECONDSWIPE="RESTART"
SECONDSWIPEPAUSE="2"
SECONDSWIPEPAUSECONTROLS="ON"
AUDIOIFACENAME="Speaker"
AUDIOIFACEACTIVE="0"
VOLUMEMANAGER="mpd"
AUDIOVOLCHANGESTEP="3"
AUDIOVOLMAXLIMIT="100"
AUDIOVOLMINLIMIT="1"
AUDIOVOLSTARTUP="0"
AUDIOVOLBOOT="30"
VOLCHANGEIDLE="TRUE"
IDLETIMESHUTDOWN="0"
POWEROFFCMD="sudo poweroff"
SHOWCOVER="ON"
READWLANIPYN="OFF"
EDITION="classic"
LANG="en-UK"
VERSION="2.5.0 - 4101984b - master"
CHAPTEREXTENSIONS="mp4,m4a,m4b,m4r"
CHAPTERMINDURATION="600"
CMDVOLUP=""
CMDVOLDOWN=""
CMDNEXT=""
CMDPREV=""
CMDREWIND=""
CMDSEEKFORW="0011746248"
CMDSEEKBACK=""
```

Any suggestions are greatly appreciated. Thanks

**Comment by AlvinSchiller** [comment, marked as answer]
Hi, have a look at this https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/2023 if it also helps in your case :)

**Reply by Map3GOSM** [reply]
Thank you! Modifying the mpd.conf was the Answer. Sorry missed that discussion somehow before posting.
